styleguide/_themes/derek/scss/snowflakes/events.scss

```diff
@@ -0,0 +1,364 @@
+.events-banner {
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/backgrounds/solve-banner.jpg');
+}
+
+// Main Events Page.
+.eventsOverview {
+  @include make-container;
+}
+
+.eventsOverview-pagination {
+  list-style-type: none;
+  margin: 0;
+  padding: 2rem 0 0;
+  text-align: center;
+}
+
+.eventsOverview-pagination-item {
+  display: inline-block;
+  list-style-type: none;
+}
+
+.eventsOverview-pagination-itemPage {
+  background: $white;
+  border: 1px solid $gray-light;
+  font-size: 1rem;
+  padding: .5rem;
+  text-decoration: none;
+
+  &:hover {
+    background: $gray-lightest;
+    color: $black;
+    text-decoration: none;
+  }
+}
+
+.eventsOverview-pagination-itemPageActive {
+  font-size: 1.4rem;
+}
+
+.eventsOverview-pagination-itemPagePrev,
+.eventsOverview-pagination-itemPageNext {
+  @include border-radius(50%);
+  background: $teal-dark;
+  border: 0;
+  color: $white;
+  padding: 0 .5rem;
+
+  &:hover {
+    background: $gray-darkest;
+    color: $white;
+  }
+}
+
+.eventsOverview-row {
+  @include make-row;
+}
+
+.eventsOverview-searchFilter {
+  @include make-sm-column(4);
+  @include make-md-column(3);
+  background: $teal-dark;
+  margin-bottom: 1rem;
+  padding-bottom: 1rem;
+  padding-top: 1rem;
```

Comment:
Well I'm trying to avoid overriding left/right padding here, but I can if you'd prefer it.